cadquery/occ_impl/shapes.py

```diff
@@ -2351,18 +2351,20 @@ def chamfer(
 
     def shell(
         self: Any,
-        faceList: Iterable[Face],
+        faceList: Optional[Iterable[Face]],
```

Comment:
Empty list evaluates to false - not need to make this Optional.

Reply:
Indeed, but that wasn't why I added the Optional. The way this function was originally written it would have worked with `faceList=None`. Later it was typehinted as Iterable.

The way I wrote it in my first commit `faceList=None` would have raised a TypeError at `for f in faceList:`. I thought it safer to handle `None` incase there was any user doing it already.

Happy to just go back to my first commit if you are.

Reply:
OK, sounds good